ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -0,0 +1,170 @@
+import { module, test } from 'qunit';
+import { v4 as uuidv4 } from 'uuid';
+import { setupApplicationTest } from 'vault/tests/helpers';
+import authPage from 'vault/tests/pages/auth';
+import {
+  createPolicyCmd,
+  deleteEngineCmd,
+  mountEngineCmd,
+  runCmd,
+  createTokenCmd,
+  mountAuthCmd,
+} from 'vault/tests/helpers/commands';
+import {
+  adminPolicy,
+  dataPolicy,
+  metadataListPolicy,
+  metadataPolicy,
+} from 'vault/tests/helpers/policy-generator/kv';
+import { writeSecret } from 'vault/tests/helpers/kv/kv-run-commands';
+
+/**
+ * This test set is for testing the flow for creating new secrets and versions
+ */
+module('Acceptance | kv-v2 workflow | secret and version create', function (hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.backend = `kv-workflow-${uuidv4()}`;
+    await authPage.login();
+    await runCmd(mountEngineCmd('kv-v2', this.backend), false);
+    await writeSecret(this.backend, 'app/first-secret', 'foo', 'bar');
```

Comment:
Each test file is going to have a different main module setup, but I left this on each one for now